rpcs3/util/sysinfo.cpp

```diff
@@ -409,13 +409,18 @@ u64 utils::get_total_memory()
 
 u32 utils::get_thread_count()
 {
+	static u32 count = []()
```

Comment:
making this static is probably a bad idea, as users can limit thread affinity at runtime...

Reply:
Just thread affinity is a bit different thing, I doubt it can affect current CPU count.

Reply:
For now it's static.